business/proxy_logging.go

```diff
@@ -0,0 +1,46 @@
+package business
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/log"
+	"github.com/kiali/kiali/util/httputil"
+)
+
+const (
+	envoyAdminPort = 15000
+)
+
+type ProxyLoggingService struct {
+	k8s kubernetes.ClientInterface
+}
+
+func (in *ProxyLoggingService) SetLogLevel(ctx context.Context, namespace, pod, level string) error {
```

Comment:
I'd recommend to place the low level logic in the kubernetes/istio.go just to have in that package the logic to define the Envoy ports, endpoints and portforwarding

![image](https://user-images.githubusercontent.com/1662329/132379742-99dbe930-f31e-4bc7-bf80-d841aa6b6610.png)

Nothing wrong, just trying to put it in the sample place for future optimizations.